src/app/containers/Header/index.jsx

```diff
@@ -8,7 +8,7 @@ const HeaderContainer = () => {
 
   return (
     <header role="banner">
-      <BrandContainer isHeader />
+      <BrandContainer {...(pageType !== 'frontPage' && { isHeader: true })} />
```

Comment:
Haha this is very clean, but hard to read, can it be a variable above please?

Reply:
Done, but it will have to be updated once [this](https://github.com/bbc/simorgh/pull/2146) is merged.